tf_trees/demo.py

```diff
@@ -0,0 +1,30 @@
+from tensorflow import keras
+
+# Import the layer.
+# If the current working directory is not tf_trees, then uncomment the following
+# lines and change "/path/to/" to the parent directory of tf_trees.
+# import sys
+# sys.path.insert(1, '/path/to/tf_trees')
+from tel import TEL
+# The documentation of TEL can be accessed as follows
+?TEL
+
+# We will fit TEL on the Boston Housing regression dataset.
+# First, load the dataset.
+from keras.datasets import boston_housing
+(x_train, y_train), (x_test, y_test) = boston_housing.load_data()
+
+# Define the tree layer; here we choose 10 trees, each of depth 3.
+# Note output_logits_dim is the dimension of the tree output.
+# output_logits_dim = 1 in this case, but should be equal to the
+# number of classes if used as an output layer in a classification task.
+tree_layer = TEL(output_logits_dim=1, trees_num=10, depth=3)
```

Comment:
maybe say "units" instead of "output_logits_dim" to be consistent with keras.layers.Dense?

Reply:
This is a subtle point. We provide the option "sum_outputs" to either sum all the tree outputs or return each tree output separately. This option can make the meaning of "units" ambiguous..